schemas/EGA.ArrayExperiment.json

```diff
@@ -0,0 +1,109 @@
+      },
+      "assayed_molecule": {
+        "type": "string",
+        "title": "Assayed molecule in the ArrayExperiment [EFO:0002772]",
+        "description": "Type of molecule assayed, chosen from a list of CV (e.g. DNA assay [EFO:0001456] or RNA assay [EFO:0001457]).",
+        "enum": ["DNA assay [EFO:0001456]", "RNA assay [EFO:0001457]", "Metabolomic profiling [EFO:0000752]", "Protein assay [EFO:0001458]"]
+      },
+      "ADF_files": {
+        "type": "array",
+        "title": "Array Design Format (ADF) [NCIT:C172213] file block",
+        "description": "The array design format (ADF) [NCIT:C172213] is the unique set of probes (with their coordinates) found on the microarray chip. They can be standard (sold by a company) or custom. Its format is of a spreadsheet-like tab-delimited text file with metadata header rows, followed by a multi-column table of probe information. This object shall only be used if the EGA is the one storing such file. Otherwise, the Relationships object shall be used to point from the ArrayExperiment to the ADF submitted elsewhere. #! Using an empty item list but definying the file object as possible additionalItems we create the correct constraint: anything but a file object is rejected, but we can add as many as we want.",
+        "additionalProperties": false,
+        "additionalItems": { "$ref": "https://github.com/EbiEga/ega-metadata-schema/tree/main/schemas/EGA.common-definitions.json#/definitions/file_object" },
+        "items": []
+      },
+      "sample_number": {
+        "type": "integer",
+        "title": "Number of samples of the ArrayExperiment",
+        "description": "Number of samples included in the experiment. One sample corresponds to one biological replicate [EFO:0002091] (it could be the genetic content from a single cell, a tissue… from a single individual or from several individuals). Shall not be mistaken for technical replicates [CHEBI:24432] being used several times (see https://www.ebi.ac.uk/seqdb/confluence/pages/viewpage.action?spaceKey=EGA&title=Sample+Representation)."
+      },
+
+      "ArrayExperiment_relationships": {
+        "type": "array",
+        "title": "ArrayExperiment relationships",
+        "description": "Comprises metadata (e.g. Source or Target) of a directional association between two entities. This relationshipos node contains all the possible relationships between metadata objects, both outside of (e.g. an Array Design Format that was submitted to ArrayExpress being linked to their microarray data within EGA) and within (e.g. an ArrayExperiment being linked to a Sample) the EGA. #! Using an empty item list but definying the relatiomship as possible additionalItems we create the correct constraint: anything but a relationship object is rejected, but we can add as many as we want.",
```

Comment:
Typo: it's written `relationshipos` and I guess the word is `relationships`

Reply:
You write: `we can add as many as we want` I rather use something like: `EGA`, `the user` other than a pronoun

Reply:
Well spotted, in fact there is another typo in the same line. When I wrote the schemas the typo-checker was disabled, so I expect some other typos to appear. 

Regarding the pronoun used, I will also address it in my next commit.

Reply:
also the "defyning the relatiomship" > "defining the relationship" part